Resources/Prototypes/Entities/Structures/Power/smes.yml

```
@@ -113,3 +116,48 @@
+    sourceNode: input
+    loadNode: output
+    collectionName: smes
+    sprite: Structures/Power/smes.rsi
+    state: advancedsmes-static
+
+- type: entity
+  parent: SMESAdvanced
+  id: SMESAdvancedEmpty
+  suffix: Empty
+  components:
+  - type: Battery
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)